OnlineFlightSearchAPI/Startup.cs

```diff
@@ -24,6 +25,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<ISearchFlightService, FlightService>();
             services.AddScoped<IAirportServices, AirportServices>();
             services.AddScoped<IFlightRepository, FlightRepository>();
+            services.AddScoped<IAirportRepository, AirportRepository>();
```

Comment:
need scoped?
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-2.2#service-lifetimes

Reply:
Removed Dependency Injection and used Mocking